packages/plugin-git/sources/gitUtils.ts

```diff
@@ -266,7 +266,7 @@ export async function resolveUrl(url: string, configuration: Configuration) {
     }
   };
 
-  return `${repo}#${resolve(protocol, request)}`;
+  return normalizeRepoUrl(`${repo}#${resolve(protocol, request)}`);
```

Comment:
I noticed while working on this PR that it wasn't working on our repository at work. Digging a bit, I found out that the assumption introduced in the PR adding support for showing the +/- cache size (namely, that all locators going into the `fetchPackageFromCache` method were also in `storedPackages`) wasn't actually true for git dependencies: the GitFetcher was normalizing the locator before passing it forward, so it was different from what the `storedPackages` object contained.

I considered a couple of ways to solve this (for example to remove this assumption, and make the cache store the locators rather than just their hash), but in the end I felt the cleanest was to simply normalize the locators directly from within the resolvers.